micro-deps/micro-deps-jax-rs/src/main/java/com/ofg/infrastructure/jaxrs/JerseyServiceResolver.java

```diff
@@ -0,0 +1,44 @@
+package com.ofg.infrastructure.jaxrs;
+
+import com.google.common.base.Supplier;
+import com.ofg.infrastructure.discovery.ServicePath;
+import com.ofg.infrastructure.discovery.ServiceResolver;
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.client.proxy.WebResourceFactory;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import java.net.URI;
+
+@Named
+public class JerseyServiceResolver implements JaxRsServiceResolver {
+    private final ServiceResolver resolver;
+    private final ClientConfig config;
+
+    @Inject
+    public JerseyServiceResolver(ServiceResolver resolver, ClientConfig config) {
+        this.config = config;
+        this.resolver = resolver;
+    }
+
+    @Override
+    public <T> Supplier<T> getLocator(final Class<T> clazz) {
```

Comment:
We should add as second parameter ServiceAlias rather than relay on package of class.